src/components/ListsTable/ListsTable.tsx

```diff
@@ -98,6 +99,8 @@ export const ListsTable: FC<ListsTableProps> = ({
         interactive
         data-testid="ItemsList"
         contentData={content ?? []}
+        // @ts-ignore:next-line
```

Comment:
Why are we ignoring this?

Reply:
+1
This has come up in the last few tech council meetings, and the consensus seems to be that when we ignore a warning, we should put add comment as close to it as possible (like the line right above the `@ts-ignore` or `@SuppressWarning`) explaining why the warning is being ignored.